src/v3/test/integration/identify-with-webauthn-autofill.test.tsx

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { setup } from './util';
+import mockResponse from '../../src/mocks/response/idp/idx/introspect/identify-with-webauthn-autofill.json';
+
+describe('identify-with-webauthn-autofill', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it('should have "username webauthn" as the autocomplete attribute on the identifier field if passkeys are available in the browser', async () => {
+    global.PublicKeyCredential = {
+      isConditionalMediationAvailable: async () => true,
+    };
```

Comment:
Make sure you reset the value of `global.PublicKeyCredential` after each test. One way to do this is to store the existing value in a global variable and then restoring it to that in the afterEach hook